doc/whats_new.rst

```diff
@@ -4,6 +4,5758 @@
 ===============
 Release History
 ===============
+
+Version 0.20 (under development)
```

Comment:
You want to add your entry to v0.20.0.rst (or something like this) rather than change whats_new.rst.